README.md

```diff
@@ -6,62 +6,29 @@ This repository is for active development of the Azure SDK for C++. For consumer
 
 ## Getting started
 
-For the best development experience, we recommend developers use [CMake projects in Visual Studio](https://docs.microsoft.com/cpp/build/cmake-projects-in-visual-studio?view=vs-2019) to view and build the source code together with its dependencies. You can also use any other text editor of your choice, such as [VS Code](https://code.visualstudio.com/), along with the command line for building your application with the SDK.
+The Azure SDK for C++ is compatible with a number of different development environments and tools. The following instructions will utilize [Visual Studio](https://visualstudio.microsoft.com/) or [VSCode](https://code.visualstudio.com/) as the IDE, [CMake](https://cmake.org/) for build automation, and [vcpkg](https://vcpkg.io/) as our package manager.
 
-You can find additional information for specific libraries by navigating to the appropriate folder in the `/sdk` directory. See the **README.md** file located in the library's project folder, for example, the [Azure Storage client library](https://github.com/Azure/azure-sdk-for-cpp/tree/main/sdk/storage#azure-storage-libraries-for-c).
+### Requirements
 
-For API reference docs, tutorials, samples, quick starts, and other documentation, go to [Azure SDK for C++ Developer Docs](https://azure.github.io/azure-sdk-for-cpp).
+- An Azure subscription. Sign up for a [free trial](https://azure.microsoft.com/pricing/free-trial/) or use your [MSDN subscriber benefits](https://azure.microsoft.com/pricing/member-offers/msdn-benefits-details/).
 
-### Download & Install the SDK
+### Development environment and tools set up
 
-Here are some alternatives, from easiest to advanced, how you can get, build and integrate Azure SDK clients to your application.
+- [Visual Studio, CMake and vcpkg](https://learn.microsoft.com/en-us/vcpkg/get_started/get-started-vs)
```

Comment:
> For Visual Studio, you should probably include the "Native Development using C++" workload.

The linked vcpkg doc's prerequisites section links to docs on installing the CMake component for Visual Studio. Is the "Native Development using C++ workload component different and includes needed or recommended pieces the "Desktop development with C++" component doesn't include? 

> FOr Visual Studio Code, you should probably list the VS Code extensions that are needed to build (The recommended C++ extensions). 
> For Visual Studio Code, you should also include instructions on how to find a C++ compiler, since VS Code doesn't include one. If you tickle things correctly, it will point the customer to the Visual Studio community edition msbuild package, but we should be explicit. We also should include links to clang/gcc as options in addition to visual studio.

Similar to my reply for VS, the linked vcpkg docs go over installing the C/C++ language extension and the CMake Tools extension. And the prerequisite section links to docs on obtaining a C++ compiler that has instructions for MSVC and MinGW for windows and GCC for Linux and Clang for macOS. 

@LarryOsterman, what's your general opinion for the main README of our repo? Should we try to show only our recommended way of getting started, the link to sperate pages for more options or specific scenarios? Or should we include all details and options in the main README directly.

Reply:
Added a point on the prereqs section that specifically calls out the need for a C++ compiler. Let me know if you think it needs any more clarification.